### ISSUE TITLE
Options

### ISSUE THREAD

**Issue by kempeitai** [raptorswing/MouseToJoystick#7]
Could you please add option for toggle key?
Double right click sometimes breaks the gameplay if you need to target with button 2.

**Comment by troopi** [comment]
Indeed. Raptorswing, this app works greatly. I'm using it with Wing Commander series.
But the use of double click on second mouse key makes it impossible to use the left mouse key in any game without messing it all up. A toggle mode with capslock or backspace would be better.
I would advise the use of a .ini text file for the app to read values from, this way, the user coul change other things also, like deadzone, mouse spring power, and sensitivity.

**Comment by raptorswing** [comment]
I don't have much time for working on this lately but I'll try to pull something together soon. I'm happy that you have found it useful.

**Comment by raptorswing** [comment]
Hello. Sorry for the extreme delay. I've created a new beta version which doesn't require the mouse-centering hack to function. It's available [here](https://github.com/raptorswing/MouseToJoystick/releases/tag/2.0-beta). Feel free to give it a try.

**Comment by tahzib1451** [comment]
hey raptor, i like what you have done here in order to play your favorite game. i can use the beta version with CFS3 which is great...just a suggestion - can you add mouse wheel as the throttle and an option to hold right mouse button and moving the mouse horizontally to act as rudder axis? as of now there is only X and Y axis so i believe these additional Z axis and RZ will bring more life to the application...
cheers
